docs/content/_terraform/modules.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Terraform Modules
+short_title: Modules
+order: 1
+---
+
+# Contents
+{:.no_toc}
+
+* This will become the Table of Contents
+{:toc}
+
+# Terraform Modules
+
+These modules simplify and standardize the creation of application infrastructure. You reference these modules from 
+`.tf` files inside your project repo via versioned URL.
+
+## Module Structure
+
+Our terraform modules follow (for the most part) the 
+[module structure](https://www.terraform.io/docs/modules/index.html) defined by Terraform. See 
+[the terraform module template](https://github.com/byu-oit/terraform-module-template) for the basic structure of our 
+terraform modules.
+
+## [Module Sources](https://www.terraform.io/docs/modules/sources.html)
+
+When using a terraform module you specify a `source` which tells Terraform where to find the source code for the module.
+
+We have opted to use the [GitHub source](https://www.terraform.io/docs/modules/sources.html#github) method using `git` 
+over HTTPS to pull the source code of our modules living in public GitHub repos. This is the easiest way to pull modules
+from GitHub from within a CICD pipeline (aka CodeBuild) without having to do extra configuration. The downside to this 
+is if you need to use a module residing in a private GitHub repo, you will then have to use a GitHub access token tied
+to a user who has access to the private repo. 
+
+## Example Modules
```

Comment:
```suggestion
## Terraform Modules
```
or something like `Standard Modules` `BYU Modules`?

Reply:
Since we're bikeshedding...

- I don't mind `Example Modules`/`Examples`.
- `Terraform Modules` as a subheading under `Terraform Modules` seems strange. But maybe that's just me.
- `Standard Modules` makes it sound official, like there's some Standard™. Maybe `Common Modules`?
- `BYU Modules` makes it sound comprehensive.